### ISSUE TITLE
Segmentation Folder Missing

### ISSUE THREAD

**Issue by Tunjii10** [remaro-network/SubPipe-dataset#7]
Hi, Great work here. I am currently trying to implement some segmentation algorithms for sub sea pipe tracking and came across this wonderful dataset. However, it seems the segmentation folder containing the images and segmentation masks is no longer available in the published dataset version. perhaps it might have been available in previous versions. Please, is it possible to get this data and if so can you point me in the right direction. Thanks.

**Comment by antallaszlo011** [comment]
Hi @Tunjii10,

Thank you for reaching out!

The currently available version of SubPipe ([v3.0.0](https://doi.org/10.5281/zenodo.10808161)) includes the semantic segmentation data we have. Please note that we only used the second chunk (Chunk2) to create the segmentation data, so this is the only chunk with the corresponding folder.

In the future, we may generate more semantic segmentation data. If that happens, we will release an updated version on Zenodo.

I hope this helps! Let me know if you have any other questions. :)

**Comment by Tunjii10** [comment]
Thank you very much, I have seen the segmentation folder.

**Comment by jpcoffelt** [comment]
Just to followup on @Tunjii10's comments...

First, great work and thanks so much for sharing the dataset! Second, I can only find 107 of the 647 (image) segmentation files mentioned in (the arXiv version of) your paper. I have already checked the latest versions of SubPipe, SubPipeMini, and SubPipeMini2 on Zenodo, but do you happen to know if the rest might still be available elsewhere?

**Comment by antallaszlo011** [comment]
Hi @jpcoffelt, thanks for the helpful tip! 

You are right, we made a mistake in the latest release of SubPipe. We accidentally included only the "test" set of segmentation images and omitted the train and validation sets. We have fixed the problem, and in the new version ([v3.0.1](https://zenodo.org/records/12666132)) all 647 images and their annotations are now available.

Thank you very much for informing us about this issue. :)
